code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -102,6 +102,14 @@
 	whitelisted = "Vaurca Worker"
 	sort_category = "Xenowear"
 
+/datum/gear/uniform/gearharness
+	display_name = "gear harness"
+	path = /obj/item/clothing/under/gearharness
+	cost = 2
+	whitelisted = list("Vaurca Worker", "Vaurca Warrior", "Diona", "Industrial Frame", "Baseline Frame")
```

Comment:
Ye, this var is handled as a string in code atm. As Alberyk said, this won't work. It might be an idea to just fall in line with the above additions and list it for `whitelisted = "Vaurca Worker"`